src/main/java/com/ericsson/gerrit/plugins/eiffel/exceptions/NoSuchElementException.java

```diff
@@ -0,0 +1,25 @@
+/*
+   Copyright 2019 Ericsson AB.
+   For a full list of individual contributors, please see the commit history.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package com.ericsson.gerrit.plugins.eiffel.exceptions;
+
+public class NoSuchElementException extends Exception {
```

Comment:
Did we not agree not to have Java doc in exceptions? Not sure anyone using this would have help from java doc. But if other exceptions has, this one should as well.